data/glossary/cache.yaml

```
@@ -63,4 +63,8 @@ entries:
 
 - term: time-to-live (TTL)
   general_definition: |-
-    the duration for which a cached copy of a resource is considered valid before it needs to be refreshed or revalidated.
\ No newline at end of file
+    the duration for which a cached copy of a resource is considered valid before it needs to be refreshed or revalidated.
+
+- term: cache lock
+  general_definition: |-
+    CDN data centers, comprising numerous servers, employ a mechanism known as a "cache lock" (mutex) to prevent overloading of origin servers. This mechanism ensures that only one server can request a specific file from the origin at any given time, facilitating efficient coordination among the servers.
```

```suggestion
    cache lock (or mutex) is a mechanism employed by CDN data centers, comprising numerous servers, to prevent overloading of origin servers. This mechanism ensures that only one server can request a specific file from the origin at any given time, facilitating efficient coordination among the servers.
```
Nit: Trying to start with the term being defined, since I believe it appears a bit late in the description.